onboarding-enabler-nodejs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zowe/apiml-onboarding-enabler-nodejs",
-  "version": "2.0.0",
+  "version": "2.0.2-PR-1964-1",
```

Comment:
Why the version here is changed to PR?

Reply:
Sorry I think this commit is generated by specific release workflow. I was trying to create a build so I can test with Zowe. Will revert these changes by the workflow.

Reply:
Hey Jack no worries! I'll take a look on the rest of PR tomorrow, I've seen there is some problem on the CI testing so I'll be debugging it. Thanks for the PR!